gammapy/estimators/utils.py

```diff
@@ -226,3 +228,25 @@ def resample_energy_edges(dataset, conditions={}):
                 energy_edges.append(energy_min)
                 break
     return u.Quantity(energy_edges[::-1])
+
+
+def lc_fvar(lightcurve):
+    r"""Wrapper to utilize the '~gammapy.stats.compute_fvar' function
```

Comment:
```suggestion
    r"""Compute the light fraction variance excess of the input lightcurve.
    
      Internally calls the `~gammapy.stats.compute_fvar` function
```
Or copy more of the initial function docstring?